lib/erb_lint/linters/deprecated_classes.rb

```diff
@@ -43,10 +44,11 @@ def lint_lines(lines)
 
       def generate_errors(class_name, line_number)
         violated_rules(class_name).map do |violated_rule|
-          message = "Deprecated class `%s` detected matching the pattern `%s`. %s#{' ' + @addendum if @addendum}"
+          suggestion = " #{violated_rule[:suggestion]}".rstrip
+          message = "Deprecated class `%s` detected matching the pattern `%s`.%s #{@addendum}".strip
           {
             line: line_number,
-            message: format(message, class_name, violated_rule[:class_expr], violated_rule[:suggestion])
+            message: format(message, class_name, violated_rule[:class_expr], suggestion)
           }
         end
```

Comment:
`split.join(' ')` will do what `strip` does plus it will take care of removing extra blank spaces between words.